components/brave_wallet/zcash/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
```

Comment:
this dep should be added to third_party/rust/vendor/Cargo.toml and the dep for `orchard` should be removed from it

Reply:
Fixed